src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10Driver.cs

```diff
@@ -14,6 +14,7 @@ public class Windows10Driver : GpioDriver
 {
     private static readonly WinGpio.GpioController s_winGpioController = WinGpio.GpioController.GetDefault();
     private readonly Dictionary<int, Windows10DriverPin> _openPins = new Dictionary<int, Windows10DriverPin>();
+    private readonly Dictionary<int, PinValue> _pinValues = new Dictionary<int, PinValue>();
```

Comment:
why does LibGpiodDriver need ConcurrentDictionary while this one is ok with regular Dictionary?

Reply:
(it's an existing issue anyway so don't worry about this in this PR)